airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -466,6 +466,8 @@ public interface Configs {
    */
   String getJobKubeMainContainerImagePullSecret();
 
+  Optional<String> getCustomImagePullSecret();
```

Comment:
I think instead of adding a brand new config, it would make more sense for our OSS users if we instead updated the existing config to support a list.

For an OSS user who wants to use custom images, they're supposed to use the [existing config,](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java#L464-L467) so having a second config will add confusion.

Basically, we just need to add support for defining multiple image pull secrets, so that Cloud can use two private image repositories instead of being limited to one.

Does that make sense?

Reply:
Makes sense, good call! I'll fix this.

Reply:
Thanks! Sorry to make you rework it all 😅

Reply:
Fixed - and retested on cloud, this should work! (though some other bugs were detected, PR coming in...)